owasp-suppressions.xml

```diff
@@ -7,13 +7,4 @@
     <packageUrl regex="true">^pkg:maven/org\.hypertrace\..*@.*$</packageUrl>
     <cpe>cpe:/a:grpc:grpc</cpe>
   </suppress>
-  <suppress until="2023-12-31Z">
```

Comment:
I'm more interested in getting rid of these suppressions than actually upgrading 😉